src/WorkbenchExtension.ts

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+
+export class WorkbenchExtension {
+  // tslint:disable-next-line: no-any
+  private static extension: vscode.Extension<any>|undefined;
+
+  private constructor(context: vscode.ExtensionContext) {
+    // Get extensionId from package.json
+    const packageJsonPath = context.asAbsolutePath('./package.json');
+    const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, 'utf8'));
+    const extensionId = packageJson.publisher + '.' + packageJson.name;
+
+    if (!extensionId) {
+      throw new Error('Fail to get extensionId from package.json.');
+    }
+    WorkbenchExtension.extension = vscode.extensions.getExtension(extensionId);
```

Comment:
Please consolidate with another helper function:
`getExtension(name: ExtensionName)` in Apis.ts

Reply:
To avoid RC extension name in source code apart from `.travis.yml` file, keep using `getExtensionId` API here.